internal/view/cronjob.go

```diff
@@ -67,43 +68,31 @@ func jobCtx(path, uid string) ContextFunc {
 func (c *CronJob) bindKeys(aa ui.KeyActions) {
 	aa.Add(ui.KeyActions{
 		tcell.KeyCtrlT: ui.NewKeyAction("Trigger", c.trigger, true),
-		ui.KeyShiftS:   ui.NewKeyAction("Suspend", c.setSuspend, true),
-		ui.KeyShiftU:   ui.NewKeyAction("Unsuspend", c.setUnsuspend, true),
+		ui.KeyShiftS:   ui.NewKeyAction("ToggleSuspend", c.toggleSuspend, true),
 	})
 }
 
-func (c *CronJob) setSuspend(evt *tcell.EventKey) *tcell.EventKey {
+func (c *CronJob) toggleSuspend(evt *tcell.EventKey) *tcell.EventKey {
 	paths := c.GetTable().GetSelectedItems()
 	if len(paths) == 0 {
 		return nil
 	}
 
 	c.Stop()
 	defer c.Start()
-	c.showSuspendDialog(paths, true)
+	c.showSuspendDialog(paths)
 
 	return nil
 }
 
-func (c *CronJob) setUnsuspend(evt *tcell.EventKey) *tcell.EventKey {
-	paths := c.GetTable().GetSelectedItems()
-	if len(paths) == 0 {
-		return nil
-	}
-
-	c.Stop()
-	defer c.Start()
-	c.showSuspendDialog(paths, false)
-
-	return nil
-}
-
-func (c *CronJob) showSuspendDialog(paths []string, suspend bool) {
+func (c *CronJob) showSuspendDialog(paths []string) {
+	suspended := strings.TrimSpace(c.GetTable().GetCell(c.GetTable().GetSelectedRowIndex(), c.GetTable().NameColIndex()+2).Text) == "true"
```

Comment:
@ludusrusso - Tx! we should probably check the GetCell return as it can return nil if there is no cell at that location and if so return early.